README.md

```diff
@@ -1,70 +1,99 @@
+- Run ```npm run test``` to run tests.
 
-### Code Splitting
+## API Interactions
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+Financial Modeling Prep API, this app is interacting/consuming the following endpoints:
 
-### Analyzing the Bundle Size
+- **Fetching company profile - Method(GET)** - <https://financialmodelingprep.com/api/v3/profile/AAPL?apikey=YOUR_API_KEY>
+- **Fetching Company list - Method(GET)** - <https://financialmodelingprep.com/api/v3/stock/list?apikey=YOUR_API_KEY>
+- **Fetching Stock Historical Prices - Method(GET)** - <https://financialmodelingprep.com/api/v3/historical-chart/1min/AAPL?apikey=YOUR_API_KEY>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+Take a look at the [DOCUMENTATION](https://financialmodelingprep.com/developer/docs) for further insights.
 
-### Making a Progressive Web App
+## Contributing
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+Contributions, issues, and feature requests are welcome!
 
-### Advanced Configuration
+Feel free to check the [issues page](https://github.com/mugberto/firm-profile/issues)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+  1. Fork the Project
+  2. Create your Feature Branch (`git checkout -b feature/newFeature`)
+  3. Commit your Changes (`git commit -m 'Add some newFeature'`)
+  4. Push to the Branch (`git push -u origin feature/newFeature`)
+  5. Open a Pull Request
 
-### Deployment
+## Authors
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+👤 **Hubert Mugabo**
 
-### `npm run build` fails to minify
+- GitHub: [@githubhandle](https://github.com/mugberto)
+- Twitter: [@twitterhandle](https://twitter.com/mugberto)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/hubert-mugabo-23144b6a/)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+## Acknowledgements
+
+- [ReactJS Organization](https://reactjs.org/) for the amazing [Documentation](https://reactjs.org/docs/getting-started.html) on React.
+- [Redux Organization](https://redux.js.org/) for the amazing [Redux Essentials Tutorial](https://redux.js.org/tutorials/essentials/part-1-overview-concepts) on Redux.
+- [Financial Modeling Prep](https://financialmodelingprep.com/) for the amazing API.
+
```

Comment:
- [ ] Kindly give appropriate credit to Nelson Sakwa the author of the original design, as required by the Creative in the "Acknowledgements" section of the readme file.